scenario-definitions.yaml

```diff
@@ -43,10 +54,9 @@ job_templates:
       DESKTOP: minimalx
       OPENQA_WEB_PROXY: 'nginx'
   openqa_install_multimachine:
-    <<: *common
+    <<: *common4G
     settings:
       DESKTOP: minimalx
-      QEMURAM: "4096"
```

Comment:
So specifying the RAM on job template level didn't work?

Reply:
no - for whatever reason it doesn't win against the machine. Maybe this is something specific to scenario definitions.
https://openqa.opensuse.org/tests/3875368#settings

Reply:
So + syntax wouldn't work? Maybe this should just be documented, or put in a ticket if it might be a bug.